### PR TITLE
Add missing jackson dependency

### PR DESCRIPTION
**Pull request by KochTobi**
For version 2.15.0 jackson-core is required to run.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -137,27 +138,26 @@
         <type>pom</type>
         <scope>test</scope>
       </dependency>
-      <!--If not specified, the 2.7.6 spring-boot-starter-parent dependency will introduce transitively the 1.30 version of snakeyaml, for which a multitude of security concerns are reported -->
-      <dependency>
-        <groupId>org.yaml</groupId>
-        <artifactId>snakeyaml</artifactId>
-        <version>2.0</version>
-      </dependency>
```
Comment (Steffengreiner) [inline pom.xml:145]: Why was this removed? 
If the version is not specified then sprnig-boot will use the 1.33 version for which security issues have been reported:
https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-1471

Comment (KochTobi) [reply pom.xml:145]: >2.7.6 spring-boot-starter-parent

Comment (KochTobi) [reply pom.xml:145]: we are now using 3.x

**Review by Steffengreiner — DISMISSED** [review]
LGTM 🚀

File: pom.xml (hunk 1 above)
Comment (KochTobi) [reply pom.xml:145]: I was wrong, turns out it is still pulled but now with version 1.33 (which has 1 CVE)

**Review by Steffengreiner — APPROVED** [review]
LGTM 🚀

**Pull request merged**